### PR TITLE
Id3: use icons and sort order from service

### PR DESCRIPTION
**Pull request by buoyad**
- [Go] move `identify3UpdateRow` params into a struct
- Move some rpc data transforms into `constants/tracker2`
- Store `siteIcon` as array and add `priority` with `Assertion`
- Use priority for sort order and network icons

This breaks the icons in the `tracker2` stories. r? @keybase/react-hackers

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers this is ready

**Comment by buoyad** [comment]
@keybase/react-hackers bump

**Comment by buoyad** [comment]
@keybase/react-hackers :bump:

File: shared/tracker2/assertion/index.js [hunk 1]
```diff
@@ -273,13 +273,55 @@ const getMenu = p => {
   }
 }
 
+const siteIconToSrcSet = siteIcon =>
+  `-webkit-image-set(${siteIcon
```
Comment (chrisnojima) [inline shared/tracker2/assertion/index.js]: lets just copy here and not slice in those containers

**Pull request merged**